### PR TITLE
Rename to elm-app-gen

### PR DESCRIPTION
**Pull request by MattCheely**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Build Elm App
+# Elm AppGen
```
Comment (MattCheely) [inline README.md]: I went back and forth between calling it "Elm App Gen" "Elm App Generator" and "Elm AppGen". Let me know if you prefer something different than what I picked here.  The CLI command is `elm-app-gen`, of course, but it reads oddly to always use that in the context of a document like this.

Comment (kalutheo) [reply README.md]: i think "Elm App Generator" is good it's a bit longer but explicit.  and the cli name `elm-app-gen` is great for me

Comment (MattCheely) [reply README.md]: done!

**Comment by kalutheo** [comment]
it's ok for me @MattCheely you have my approval

**Pull request merged**